docs/querying/functions.md

```diff
@@ -669,6 +669,8 @@ over time and return an instant vector with per-series aggregation results:
 * `min_over_time(range-vector)`: the minimum value of all points in the specified interval.
 * `max_over_time(range-vector)`: the maximum value of all points in the specified interval.
 * `sum_over_time(range-vector)`: the sum of all values in the specified interval.
+   Note: for a mathematically correct [numerical integration](https://en.wikipedia.org/wiki/Numerical_integration) using the [rectangle method](https://en.wikipedia.org/wiki/Riemann_sum) also multiply by a factor to expand the rectangles to take up the full time range without gaps.
+   eg. for a `_per_second` gauge metric, with time-period between samples (scrape interval) of 15s, multiply by 15: `sum_over_time(<metric_prefix>_per_second[<duration>]) * 15`
```

Comment:
When using `sum_over_time` recently I was trying to find the increase log GB ingested over a day. 
The metric that we were already working with was  `…_logs_…_bytes_received_per_second` - ie a gauge metric representing a rate. So I needed to find the area under the curve - hence why I used `sum_over_time`.

But I'm realising that, if I had access to a counter metric instead (something like `…_bytes_received_total`, then I could use the `increase()` function instead. This seems preferable, as when using `increase()` you don't need to know the scrape-interval. And missing samples won't throw the results off too.

So shall I include something along the lines of "Prefer using `increase()` with a counter metric to integrating a rate metric, as it's more robust to missing samples and changes in scrape interval". WDYT?